magento/inventory
# Add to configuration of Store pickup method radius for search results of store selection

### Description (*)

Contains https://github.com/magento/inventory/pull/2651
### Fixed Issues (if relevant)

1.https://github.com/magento/inventory/issues/2650: Add to configuration of Store pickup method radius for search results of store selection

### Manual testing scenarios (*)

1. Create two sources with store pickup.(Culver City, Austin)
2. Create simple product and assign two sources to product.
3. Add product to cart and navigate to checkout.
4. Click 'Pick in Store' try to search Culver City. 
5. Verify, only Culver City source found.
6. Navigate to admin area Store>Configuration>Sales>Delivery Methods>In-Store Delivery
7. Change search radius to 2000 or greater
8. Navigate to checkout.
9. Click 'Pick in Store' try to search Culver City. 
10. Verify, both sources for Culver City and Austin are found.

### Questions or comments


### Contribution checklist (*)
